.github/workflows/ci-post-workflow.yml

```
@@ -60,13 +60,13 @@ jobs:
             const fs = require('fs');
             return fs.readFileSync('./number', 'utf8').toString();
           result-encoding: string
-      - uses: peter-evans/find-comment@v2
+      - uses: peter-evans/find-comment@1769778a0c5bd330272d749d12c036d65e70d39d # v2.0.0
```

just curious: why not pin to `v2.0.0`?

```suggestion
      - uses: peter-evans/find-comment@v2.0.0
```

---

It's pretty simple. When you point to the tag, then in case if GHA repo will get compromised (which is out of our control), tag can be easily moved to another commit, branch etc. And we will not even get notified about such events.
With the hash such things will not work.
Yes, it might be not so convenient, but it is an eternal trade off - security vs convenience.
Also dependabot can track GHA updates by the hash as well.